src/components/SessionExpiry.jsx

```diff
@@ -10,14 +10,13 @@ const SessionExpiry = () => {
   }
 
   const displayTime = (timeLeft) => {
-    if(Math.floor(timeLeft % 60) < 10){
-      return Math.floor(timeLeft/60) + ":0" + Math.floor(timeLeft % 60)
-    } else {
-      return Math.floor(timeLeft/60) + ":" +  Math.floor(timeLeft % 60)
-    }
+    const hh = Math.floor(timeLeft/60);
+    const mm = `${Math.floor(timeLeft % 60)}`.padStart(2, "0");
+    
+    return `${hh}:${mm}`;
   }
 
-  useEffect(() => {
+  const autoLogOut = () => {
     const countDown = () => {
```

Comment:
does autoLogOut return anything? countDown is declared but is it returned?

Reply:
I can work on this later today.

Reply:
No it shouldn't return anything.  It is just used for a side effect.  Countdown is called in autoLogOut.  Although I should remove Countdown's declaration out of autoLogOut.